### DISCUSSION TITLE
Include "Ski Touring Trail" in Winter and Ski map style

### DISCUSSION THREAD

**Discussion by caddish** [osmandapp/OsmAnd#13824] [General]
Currently the feature type : "Ski Touring Trail" in OSM doesn't show in OsmAnd "Winter and Ski" map style (see exemple below) or any other OsmAnd map style

I got lost this weekend and when I went on OSM to add the trail noticed it was already there ...

Can I manually add this type of way to OsmAnd or it's hard coded ? 

https://www.openstreetmap.org/way/857804964#map=17/45.08104/-71.58932&layers=C

**Comment by xmd5a2** [comment]
It is rendered as on screenshot
![Screenshot_2022-03-05_15-55-31](https://user-images.githubusercontent.com/5829014/156884009-cc1e95bd-0c08-482d-b540-c347e4ea0785.png)

**Reply by caddish** [reply]
My bad, I forgot to update OsmAnd, now I wish I did this before I got lost in the dark :P

![Screenshot_20220306-090325_Strava](https://user-images.githubusercontent.com/3298202/156926694-3bda6730-8700-4135-99f3-dbac04b4568d.jpg)

**Reply by Poloxo53** [reply]
Hi,
@xmd5a2 could you tell me how did you customize the rendering in Osmand ? Yours looks like Elevate style from openandromaps, am I right ?
